src/libraries/Microsoft.PowerFx.Interpreter/Environment/PowerFxConfigExtensions.cs

```diff
@@ -37,7 +37,6 @@ public static void EnableSetFunction(this PowerFxConfig powerFxConfig)
         public static void EnableMutationFunctions(this SymbolTable symbolTable)
         {
             symbolTable.AddFunction(new RecalcEngineSetFunction());
-            symbolTable.AddFunction(new CollectFunction());
```

Comment:
>   symbolTable.AddFunction(new CollectFunction()); [](http://example.com/codeflow?start=9&length=50)

Does this mean Collect is now always part of the default base set?

Reply:
This was changed with PA in mind since mutations are active by default but we probably should keep these deactivated by default and enable them in PA.

Reply:
Collect will remain disabled by default.